service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh

```diff
@@ -76,9 +76,15 @@ fi
 # Log in with app-default-credentials
 sudo -u "${JUPYTER_USER}" sh -c "terra auth login --mode=APP_DEFAULT_CREDENTIALS"
 
-# Get all the git repvoio clone urls from the workspace resource list with type GIT_REPO and try to clone all of them.
+sudo -u "${JUPYTER_USER}" sh -c "mkdir -p /home/${JUPYTER_USER}/.ssh"
 cd /home/${JUPYTER_USER}
-sudo -u "${JUPYTER_USER}" sh -c "terra resource list --type=GIT_REPO --format=JSON" | jq -r 'unique_by(.gitRepoUrl) | .[] | {gitRepoUrl} | join (" ") ' | while read -r repoUrl; do
-  sudo -u "${JUPYTER_USER}" sh -c "git clone ${repoUrl}" && echo "cloned ${repoUrl}" || echo "clone ${repoUrl} failed"
-done
+readonly TERRA_SSH_KEY=$(sudo -u "${JUPYTER_USER}" sh -c "terra user ssh-key get --format=JSON")
```

Comment:
Does this error out if there's no SSH key, or just return an empty structure?

Reply:
it returns empty